### ISSUE TITLE
Separate splice mutations from truncation mutaitons

### ISSUE THREAD

**Issue by jjgao** [cBioPortal/icebox#80]
I think we should render splice mutations in its own category. For example in [this query](http://www.cbioportal.org/beta/results/plots?session_id=5bd3ccdf498eb8b3d56887fb), the samples with TP53 truncating mutations are all very low except for the splice mutations.

![image](https://user-images.githubusercontent.com/840895/47598714-2d091500-d96e-11e8-8d5d-5376c6bd4592.png)

- define color for splice mutations 
- separate in oncoprint
- separate in mutations tab
- oql
- We may also want to do that in OncoKB -- will discuss with OncoKB team.

@schultzn @tmazor thoughts?

**Comment by schultzn** [comment]
Agreed, that would make sense. A new category is easy - but finding a new color for OncoPrints would be harder (but maybe also useful).

**Comment by jjgao** [comment]
@schultzn you new comment is conflicting with your [previous one](https://github.com/cBioPortal/icebox/issues/80#issuecomment-434696537). If you think this is a good idea, I'll move the issue back to backlog.
